src/arch/x86/regs/float.hh

```diff
@@ -179,6 +213,24 @@ xmmHigh(int index)
     return floatRegClass[XmmBase + 2 * index + 1];
 }
 
+static inline RegId
+ymm(int index)
+{
+    return floatRegClass[YmmBase + index];
+}
```

Comment:
This function is unused. Unless there is a good reason to keep it, it should be removed.